autospotting.go

```diff
@@ -30,7 +30,8 @@ func main() {
 func run() {
 	log.Println("Starting autospotting agent, build ", Version, "expiring on", ExpirationDate)
 
-	if isExpired(ExpirationDate) {
+	if false {
```

Comment:
Please don't change the expiration logic.

Reply:
Will change this.